backend/app/assets/stylesheets/spree/backend/sections/_style_guide.scss

```diff
@@ -122,6 +122,10 @@
 .style-guide-code,
 .style-guide-result {
   margin: 1rem 0;
+
+  img {
+    max-width: none;
+  }
```

Comment:
Why is this change necessary?

Reply:
This shows the image under style_guide page
![Screenshot 2020-06-26 at 16 35 22](https://user-images.githubusercontent.com/387690/85868812-17936080-b7cb-11ea-9c41-5d2d51648628.png)

Without it, the image doesn't appear
![Screenshot 2020-06-26 at 16 35 06](https://user-images.githubusercontent.com/387690/85868794-12ceac80-b7cb-11ea-9548-42fb552e0199.png)